frontend/src/lib/api.ts

```diff
@@ -1537,6 +1537,12 @@ const api = {
             return await new ApiRequest().recording(recordingId).withAction('persist').create()
         },
 
+        async summarize(
+            recordingId: SessionRecordingType['id']
+        ): Promise<{ content: string; ai_result: Record<string, any> }> {
+            return await new ApiRequest().recording(recordingId).withAction('summarize').create()
```

Comment:
What's about data privacy? Is everything being sent to OpenAI?
I believe this should be disabled by default with a huge note when enabling it.

Reply:
Absolutely... This is behind a flag and not production ready

Right now we only send event names, urls, and a text from som HTML elements.

Reply:
console logs and network requests (no headers or payload) would be super useful as well.

Reply:
> text from som HTML elements

That's what I am afraid of, even if we use only within our org, if the text form contains an email/name, it is already a problem, we'd need something like [this](https://docs.sentry.io/product/issues/issue-details/ai-suggested-solution/#data-processing).